### PR TITLE
Updated changelog.md for 0.8.x

### PR DESCRIPTION
**Pull request by sjsadowski**
Documented all fixes and changes in the changelog with attributions to the PR authors

### CODE REVIEW

**Comment by seemethere** [comment]
Great job! We should probably find some way to automate this in the future

**Comment by sjsadowski** [comment]
@seemethere once I pull the contributors guide together, I think that'll be pretty straightforward.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,99 @@
...
+    - Fix: Exception for missing parameter when value is null (NyanKiyoshi)
+    - Fix: Parameter check (Howie Hu)
+    - Fix (1089): Routing issue with named parameters and different methods (yunstanford)
+    - Fix (1085): Signal handling in multi-worker mode (yunstanford)
+    - Fix: single quote in readme.rst (Cosven)
+    - Fix: method typos (Dmitry Dygalo)
+    - Fix: log_response correct output for ip and port (Wibowo Arindrarto)
+    - Fix (1042): Exception Handling (Raphael Deems)
+    - Fix: Chinese URIs (Howie Hu)
+    - Fix (1079): timeout bug when self.transport is None (Raphael Deems)
```
Comment (r0fls) [inline CHANGELOG.md]: Sorry to be a bother, but could you change this to `Raphael Deem`?

**Comment by sjsadowski** [comment]
@r0fls Sorry about that - corrected in my fork.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,22 +1,99 @@
...
+    - Fix (1050): add samesite cookie to cookie keys (Raphael Deem)
+    - Fix (1065): allow add_task after server starts (Raphael Deem)
+    - Fix (1061): double quotes in unauthorized exception (Raphael Deem)
+    - Fix (1062): inject the app in add_task method (Raphael Deem)
+    - Fix: update environment.yml for readthedocs (Eli Uriegas)
+    - Fix: Cancel request task when response timeout is triggered (Jeong YunWon)
+    - Fix (1052): Method not allowed response for RFC7231 compliance (Raphael Deem)
+    - Fix: IPv6 Address and Socket Data Format (Dan Palmer)
+
+Note: Changelog was unmaintained between 0.1 and 0.7
```
Comment (TheKevJames) [inline CHANGELOG.md]: FYI: looks like the changelog was maintained in the [GitHub releases](https://github.com/huge-success/sanic/releases) between these versions. It'd be great if we could pull those into this doc.

Comment (sjsadowski) [reply CHANGELOG.md]: Given that the original maintainers chose not to for some reason, I don't know that we need to go back and step on toes.

**Pull request merged**